galaxy_api/api/ui/serializers/collection.py

```diff
@@ -70,7 +70,7 @@ class CollectionVersionBaseSerializer(serializers.Serializer):
 
 
 class CollectionLatestVersionSerializer(CollectionVersionBaseSerializer):
-    metadata = CollectionMetadataBaseSerializer(source='*')
+    metadata = CollectionMetadataSerializer(source='*')
```

Comment:
CollectionMetadataSerializer includes `dependencies` and `contents` fields. Are you sure that this is desired behaviour?

Reply:
The serializers were kind of a mess, so I took the chance to clean it up a little bit.